tests/integration/federation/test.sh

```diff
@@ -20,9 +23,19 @@ TRUST_DOMAIN_2=${TRUST_DOMAIN_2:-td2}
 NAMESPACE_POLICY_NAMESPACE=${NAMESPACE_POLICY_NAMESPACE:-demo}
 POD_POLICY_POD_LABEL=${POD_POLICY_POD_LABEL:-"foo=bar"}
 
-function configure() {
+function init() {
   rm -f cofide.yaml
-  ./cofidectl init
+  args=""
+  if [[ -n "$DATA_SOURCE_PLUGIN" ]]; then
+    args="$args --data-source-plugin $DATA_SOURCE_PLUGIN"
+  fi
+  if [[ -n "$PROVISION_PLUGIN" ]]; then
+    args="$args --provision-plugin $PROVISION_PLUGIN"
+  fi
+  ./cofidectl init $args
```

Comment:
Is there anything useful we can assert on the init'd CLIs, or happy that the rest of the tests pass without incident?

Reply:
Yeah I think that could be a simple but useful test to verify the flag works beyond the rest of the tests passing (provided it's not a headache to add)

Reply:
Good call, added.